equinix/provider_test.go

```diff
@@ -127,6 +128,11 @@ func init() {
 	testAccProviders = map[string]*schema.Provider{
 		"equinix": testAccProvider,
 	}
+	testAccProviderFactories = map[string]func() (*schema.Provider, error){
```

Comment:
Can you link to docs about what this change does?  Will we eventually have only one of `testAccProviders` or `testAccProviderFactories`, or will we always need both?

Reply:
From the official testing lib doc, providers is deprecated and providerFactories needs to be use from now on.
https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/helper/resource#TestCase

We need a bug/enhancement task to use`testAccProviderFactores` instead of `testAccProviders`. `goodFirstIssue`